### DISCUSSION TITLE
Help with Building a Use Case and MVP for Deepgram’s Medical Model

### DISCUSSION THREAD

**Discussion by formai-solutions** [deepgram/community#925] [General help] [answered]
Hi everyone, I’m Robert!

I’m working on demonstrating Deepgram’s Nova-2 Medical model for Pro-Med Grupo Mexico. The focus is on using real-time transcription to enhance clinical documentation. However, I’m not a coder, and I’m looking for guidance on how to start building a minimum viable product (MVP) or any use case examples that might help me get started.

Any advice or resources would be much appreciated!

Thanks in advance!

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
@formai-solutions Hi Robert! One option is to reuse our demo at https://deepgram-conversational-demo.vercel.app/. That app has the voicebot talk about Deepgram. The code is open-source, so if you're able to get into some technical work, you can clone the [repository](https://github.com/deepgram-devs/deepgram-ai-agent-demo), and change the system prompt [here](https://github.com/deepgram-devs/deepgram-ai-agent-demo/blob/ddbcdd8c03f4310e641b4de11b1f45e34085520e/app/lib/constants.ts#L3-L81), as well as switching from Nova-2 to Nova-2 Medical [here](https://github.com/deepgram-devs/deepgram-ai-agent-demo/blob/ddbcdd8c03f4310e641b4de11b1f45e34085520e/app/context/Deepgram.tsx#L38). Then you can run the edited version on your computer as a customized demo.

Another option is to log into the Deepgram Console, go to "Demo: Transcribe your voice live" near the bottom, "Start Mission", and then you can use our pre-built demo to show streaming transcription (not a voicebot). That will use our Nova-2 model.

Finally, you can upload files for pre-recorded transcription on our [Playground](https://playground.deepgram.com/?endpoint=listen&smart_format=true&language=en&model=nova-2-medical). You can enter in model names, including `nova-2-medical`, to run tests on the Nova-2 Medical model. That won't be for a streaming use-case.
